README.rst

```diff
@@ -23,6 +23,7 @@ found in ``COPYING``.
 
 How to build this documentation
 -------------------------------
+Find the full documentation for the [Ubuntu Packaging Guide]("https://canonical-ubuntu-packaging-guide.readthedocs-hosted.com/en/latest/) at "Read the Docs".
```

Comment:
Hi, thanks for taking the time to improve the Ubuntu Packaging Guide!  

The README is written in the [reStructuredText format](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html) and not in markdown, like most other repositories on GitHub.

This would be the correct link format in reStructuredText:
```suggestion
Find the full documentation for the `Ubuntu Packaging Guide <https://canonical-ubuntu-packaging-guide.readthedocs-hosted.com/en/latest/>`_ at "Read the Docs".
```